src/components/BenthicPitLitObservationSummaryStats/BenthicPitLitObservationSummaryStats.jsx

```diff
@@ -9,47 +9,58 @@ import { ObservationsSummaryStats, Td, Th, Tr } from '../generic/Table/table'
 const BenthicPitLitObservationSummaryStats = ({
   benthicAttributeSelectOptions,
   observations = [],
+  transectLengthSurveyed = null
 }) => {
   const observationTopLevelAttributeCategoryOccurance = useMemo(() => {
-    const totalNumberOfObservations = observations.length
+    const transectLengthSurveyedInCm = transectLengthSurveyed * 100
 
     const getBenthicAttributeById = (benthicAttributeId) =>
       benthicAttributeSelectOptions.find((benthic) => benthic.value === benthicAttributeId)
 
     const observationsWithTopLevelCategoryNames = observations.map((observation) => {
       const benthicAttribute = getBenthicAttributeById(observation.attribute)
-
+      const topLevelCategory = getBenthicAttributeById(benthicAttribute?.topLevelCategory)
+      
       return {
         ...observation,
-        topLevelCategoryName: getBenthicAttributeById(benthicAttribute?.topLevelCategory)?.label,
+        topLevelCategoryName: topLevelCategory?.label || 'Missing benthic attribute',
```

Comment:
Please add this as a language token
```suggestion
        benthic_observations.missing_benthic_attribute: 'Missing benthic attribute',
```